Base/QTCore/qSlicerCoreIOManager.cxx

```diff
@@ -1133,6 +1133,18 @@ void qSlicerCoreIOManager::registerIO(qSlicerIO* io)
     }
 }
 
+//-----------------------------------------------------------------------------
+int qSlicerCoreIOManager::numberOfRegisteredFileReader(const qSlicerIO::IOFileType& fileType) const
```

Comment:
We should follow Qt conventions in Qt classes, which means that we should use `registeredFileReaderCount` name.
```suggestion
int qSlicerCoreIOManager::registeredFileReaderCount(const qSlicerIO::IOFileType& fileType) const
```